src/test/java/com/marklogic/appdeployer/DefaultAppConfigFactoryTest.java

```diff
@@ -929,6 +929,70 @@ void globalKeyStoreAndTrustStore() {
 		assertEquals("SunX5092", config.getAppServicesTrustStoreAlgorithm());
 	}
 
+	@Test
```

Comment:
You can get these all done in one test with a JUnit parameterized test. We have a few examples in the Spark connector project - e.g.

```
@ParameterizedTest
    @ValueSource(strings = {
        "mlRestPort",
        "mlAppServicesPort",
        etc...
    })
void invalidNumericValue(String propertyName) {
 ....
}
```

Reply:
baeldung for win